### PR TITLE
[Feat](configuration): Integrate pytest and be

### PR DESCRIPTION
**Pull request by Stanford997**
## Context

pytest workflow using github actions

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Python application
```
Comment (adamma1024) [inline .github/workflows/python-app.yml]: "CI for BE" is better

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+  push:
```
Comment (adamma1024) [inline .github/workflows/python-app.yml]: Please specify the folder here so as to not be disturbed by the frontend or general changes

File: .github/workflows/python-app.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        if [ -f be_repo/requirements.txt ]; then pip install -r be_repo/requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        pytest be_repo/
```
Comment (adamma1024) [inline .github/workflows/python-app.yml]: Perhaps, you can set a work_dir to "be_repo/" and then all operations will be run in it.

File: .github/workflows/python-app.yml (hunk 1 above)
Comment (Stanford997) [reply .github/workflows/python-app.yml]: Sure

File: .github/workflows/python-app.yml (hunk 2 above)
Comment (Stanford997) [reply .github/workflows/python-app.yml]: Fixed

File: .github/workflows/python-app.yml (hunk 3 above)
Comment (Stanford997) [reply .github/workflows/python-app.yml]: Amazing

**Review by adamma1024 — APPROVED** [review]
LGTM

**Pull request merged**